swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -243,77 +257,13 @@ Follow the instructions given in this section to develop the service.
 
     - Define the `hospitalEP` variable. Later, the relevant client is assigned to this variable based on the `hospital_id` value.
```

Comment:
- Define an `http:Client` variable named `hospitalEP`. Later, the relevant client is assigned to this variable based on the `hospital_id` value.